website/docs/r/fortios_routerbgp_network.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+subcategory: "FortiGate Router"
+layout: "fortios"
+page_title: "FortiOS: fortios_routerbgp_network"
+description: |-
+  BGP network table.
+---
+
+# fortios_routerbgp_network
+BGP network table.
+
+~> The provider supports the definition of Network in Router Bgp `fortios_router_bgp`, and also allows the definition of separate Network resources `fortios_routerbgp_network`, but do not use a `fortios_router_bgp` with in-line Network in conjunction with any `fortios_routerbgp_network` resources, otherwise conflicts and overwrite will occur.
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `prefix` - (Required) Prefix of network.
```

Comment:
Can you please add:
* `fosid` - ID.                      <~~~~~~~~~~~~~~~~~
* `prefix` - (Required) Prefix of network.
Used to match all fields in the terraform state file